lib/widgets/katex.dart

```diff
@@ -180,6 +176,23 @@ class _KatexSpan extends StatelessWidget {
       widget = Padding(padding: margin, child: widget);
     }
 
+    switch (styles.position) {
+      case KatexSpanPosition.relative:
+        final offset = switch (styles.topEm) {
+          final topEm? => Offset(0, topEm * em),
+          null => null,
+        };
+
+        if (offset != null) {
```

Comment:
nit: these are closely related; keep joined as one stanza
```suggestion
        };
        if (offset != null) {
```